include/aws/s3/private/s3_request.h

```diff
@@ -23,6 +24,62 @@ enum aws_s3_request_flags {
     AWS_S3_REQUEST_FLAG_ALWAYS_SEND = 0x00000004,
 };
 
+/**
+ * Information sent in the meta_request telemetry callback.
+ */
+struct aws_s3_request_metrics {
+    struct aws_allocator *allocator;
+
+    struct {
+        /* The time stamp when the request started by S3 client, which is prepared time by the client */
+        uint64_t start_timestamp_ns;
```

Comment:
Is it a time stamp? That implies I can convert this number to a calendar date and time. Is that the case or is it from the monotonic clock?

Reply:
They are from the monotonic clock, I'll add document about it, thanks!